Makefile

```diff
@@ -1,17 +1,37 @@
 REPO := quay.io/boson/faas
 BIN  := faas
 
+WINDOWS=$(BIN)_windows_amd64.exe
+LINUX=$(BIN)_linux_amd64
+DARWIN=$(BIN)_darwin_amd64
+
 CODE := $(shell find . -name '*.go')
 DATE := $(shell date -u +"%Y%m%dT%H%M%SZ")
 HASH := $(shell git rev-parse --short HEAD 2>/dev/null)
 VTAG := $(shell git tag --points-at HEAD)
 VERS := $(shell [ -z $(VTAG) ] && echo 'tip' || echo $(VTAG) )
 
-all: $(BIN)
+all: $(WINDOWS) $(LINUX) $(DARWIN)
 build: all
 
-$(BIN): $(CODE)
-	go build -ldflags "-X main.date=$(DATE) -X main.vers=$(VERS) -X main.hash=$(HASH)" ./cmd/$(BIN)
+
+windows: $(WINDOWS) ## Build for Windows
+
+linux: $(LINUX) ## Build for Linux
+
+darwin: $(DARWIN) ## Build for Darwin (macOS)
+
+$(WINDOWS):
+	env GOOS=windows GOARCH=amd64 go build -i -v -o $(WINDOWS) -ldflags "-X main.date=$(DATE) -X main.vers=$(VERS) -X main.hash=$(HASH)" ./cmd/$(BIN)
```

Comment:
This `install` flag is causing problems for me. It tries to install something to /usr where normal user cannot write. @lkingland can you please verify it's not working for you?

Reply:
Otherwise lgtm.